packages/common/src/hooks/contacts/useAllConnections.ts

```diff
@@ -22,7 +24,26 @@ export const useAllConnections = (enabled: boolean) => {
     };
 
     const allConnections = await internalGetConnections(undefined, CHUNKSIZE);
-    return allConnections.sort((a, b) => a.odinId.localeCompare(b.odinId));
+
+    const getContactDataAndUpdateCache = async (connection: DotYouProfile) => {
+      const contact = await getContactByOdinId(dotYouClient, connection.odinId);
```

Comment:
Best to change this one to use the cached data from the `useContact` at the very least; And even better to use the `fetchQuery` with the same options from react query so it fills the same cache up automatically